_news/20230920-tstrait-0.0.1.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news
+title: New software for simulating quantitative traits
+description: | 
+  Often we want to map a genotype to a phenotype. A new `tskit`-based
+  simulator leverages the succinct tree sequence format to do so
+  efficiently for large numbers of genomes.
+  
+  The new software, tstrait, takes an existing tree sequence containing
+  mutations, and simulates a quantitative phenotypic trait whose value
+  is partially determined by a selected set of causal mutations in the
+  tree sequence.
+
+  This is an initial 0.0.1 release.
+image: ../assets/images/news/20230905-tstrait/daiki.jpg
+timestamp: 2023-09-20
+author: Yan Wong
+---
+
+# Ts-trait: new software for simulating quantitative traits (phenotypes)
+
+![Daiki Tagami]({{ '/assets/images/news/20230920-tstrait-0.0.1/daiki.jpg' |  relative_url}}){:class="news-float-left"}
```

Comment:
Makes the picture too big. Try using e.g. `{:class="news-float-left" style="width: 8em"}`